src/cultionet/model.py

```diff
@@ -462,7 +510,7 @@ def fit(
             trainer.test(
                 model=lit_model,
                 dataloaders=data_module.test_dataloader(),
-                ckpt_path='last'
+                ckpt_path='best'
```

Comment:
cool, so this makes sure  the best model from the previous trainer.fit call will be loaded 👍

Reply:
Yeah, our 'last.ckpt' file is always the best model, so 'last' or 'best' should be the same here.